CSS/style.css

```diff
@@ -120,16 +120,19 @@ body {
     top: 0.2rem;
     padding-top: 60px;
     flex-direction: column;
-    background-color: #fff;
     background-size: cover;
     background-repeat: no-repeat;
-    width: 51%;
     height: 100vh;
     text-align: start;
     padding-left: 30px;
     transition: 0.3s;
     box-shadow: 0 10px 27px rgb(0 0 0 / 5%);
-    margin-left: 150px;
+  }
+
+  .leftArrow {
+    background-color: #6070ff;
+    opacity: 0.8;
+    width: 340px;
   }
 
   .navSpan {
```

Comment:
# Your Design
![student](https://user-images.githubusercontent.com/93039359/181199936-687f91d0-86ef-4903-95ea-960eae785ddd.png)
# Figma Template with Corrections
![Figma (3rd copy)](https://user-images.githubusercontent.com/93039359/181199960-8b64cb03-6859-488e-8a73-9bf1af80a25c.png)

- [ ] Please add this black bar shown in the Figma template to your design to follow the Figma template. :blush:
- [ ] Please add the backdrop filter to make your blue screen blur. :+1:
- [ ] Please make your close button on the front of the menu because it is n